### ISSUE TITLE
Update wasm endpoint for non-classic chain cosmwasm chain

### ISSUE THREAD

**Issue by y-pakorn** [terra-money/hive-graph#16]
Current `pisco-1` testnet hive is using the old Terra wasm endpoint `/terra/wasm/v1beta1/` causing wasm query to fail and return `Not Implemented` error. The non-classic chain should use the direct cosmwasm endpoint `/cosmwasm/wasm/v1/`.

This query on [testnet hive](https://pisco-hive.terra.dev/graphql) will fail with such error.

```graphql
{
  wasm {
    codeInfo(codeID: 12) {
      code_creator
    }
  }
}
```

```json
{
  "errors": [
    {
      "message": "[Not Implemented] error",
      "locations": [
        {
          "line": 3,
          "column": 5
        }
      ],
      "path": [
        "wasm",
        "codeInfo"
      ],
      "extensions": {
        "code": "INTERNAL_SERVER_ERROR",
        "exception": {
          "status": 501,
          "name": "LCDClientError",
          "message": "[Not Implemented] error",
          "stacktrace": [
            "LCDClientError: [Not Implemented] error",
            "    at WasmService.codeInfo (/app/dist/wasm/wasm.service.js:36:19)",
            "    at runMicrotasks (<anonymous>)",
            "    at processTicksAndRejections (internal/process/task_queues.js:95:5)",
            "    at async target (/app/node_modules/@nestjs/core/helpers/external-context-creator.js:77:28)"
          ]
        }
      }
    }
  ],
  "data": null
}
```

**Comment by hanjukim** [comment]
@y-pakorn It's been fixed by #18 and deployed to pisco-hive.terra.dev

**Comment by hanjukim** [comment]
```graphql
{
  wasm {
    codeInfo(codeID: 12) {
      code_creator
    }
  }
}
```
Now returns,
```graphql
{
  "data": {
    "wasm": {
      "codeInfo": {
        "code_creator": "terra1em2sanwzpv5226dxlp3rwf765r7x5gvrqfdk7p"
      }
    }
  }
}
```
As expected.
